# 0xF3 : Status Message - Software Information

# Variables declaration
	uint8 SyncByte  # IMU Address
	uint8 MessageID  # Message ID
	uint8 firmware_major_version  # Firmware Major Version
	uint8 firmware_minor_version  # Firmware Minor Version
	uint16 Checksum  # Uint16 Checksum. Complement = False
